doc/whats_new/v1.4.rst

```diff
@@ -122,6 +122,11 @@ Changelog
   together with subsampling (i.e. `max_features` < 1.0).
   :pr:`28184` by :user:`Michael Mayer <mayer79>`.
 
+- |Fix|: :class:`impute.SimpleImputer` now raises an error instead of casting
```

Comment:
Sorry I was not explicit enough :)

```suggestion
:mod:`sklearn.impute`
.....................

- |Fix|: :class:`impute.SimpleImputer` now raises an error instead of casting
```